src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/09-services.e2e-spec.ts

```diff
@@ -24,4 +24,26 @@ describe('Services page', () => {
 
     services.deleteService('mds.test');
   });
+
+  it('should create snmp-gateway service with version V2c', () => {
+    services.navigateTo('create');
+    services.addService('snmp-gateway', false, '1', 'V2c');
+    services.checkExist('snmp-gateway', true);
+
+    services.clickServiceTab('snmp-gateway', 'Details');
+    cy.get('cd-service-details').within(() => {
+      services.checkServiceStatus('snmp-gateway');
+    });
+  });
```

Comment:
Shouldn't we delete a service (or whatever) after we create it? I mean, just for saving resources. Otherwise every of these tests will just add more and more load.

Reply:
Yes, will do